test/ngResource/resourceSpec.js

```diff
@@ -97,21 +97,50 @@ describe('basic usage', function() {
     $httpBackend.flush();
   });
 
-  it('should include a request body when calling custom delete with hasBody is true', function() {
+  it('should include a request body when calling custom method with hasBody is true', function() {
+    var instant = {name: 'info.txt', value: 'V2hlbiB0aGUgdGltZSBlbmRzLg=='};
+    var fid = 42;
+    var created = {fid: fid, filname: 'fooresource', value: 'V2hlbiB0aGUgdGltZSBlbmRzLg=='};
```

Comment:
Nit: You don't really need these 2 extra variables (`fid`, `created`). You could inline them (and only keep the necessary props) to avoid visual noise.